src/AVFoundation/Enums.cs

```diff
@@ -602,7 +621,8 @@ public enum AVCaptureVideoStabilizationMode : long {
 #if XAMCORE_4_0
 	[NoMac]
 #endif
-	[NoTV, NoWatch, iOS (8,0)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[NoTV, NoWatch, iOS (8,0), Mac (10,15)]
```

Comment:
that's incorrect. Look a few lines earlier and you'll see
```csharp
#if XAMCORE_4_0
 	[NoMac]
 #endif
```

what happened is that the `enum` (which is not something that really exists after compilation for ObjC) is not used for any macOS API. As such it should not be included in the bindings [1] and it's version is meaningless.

[1] the `XAMCORE_4_0` is used so we can (in the future) build a version without the mistakes we have found (but doing so today would be a breaking change)